### PR TITLE
Slim down image and simplify readme

### PR DESCRIPTION
**Pull request by xvrl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,38 @@
...
 
 ```sh
-git clone https://github.com/druid-io/docker-druid.git
-docker build -t docker-druid docker-druid
+docker pull druidio/example-cluster
+docker run --rm -i -p 3000:8082 -p 3001:8081 -p 3090:8090 example-cluster
 ```
 
-## Run a simple Druid cluster
+Wait a minute or so for the Druid to download the sample data an start up.
```
Comment (rasahner) [inline README.md]: While this is open, can this say "for Druid to start up and download the sample data"
(remove "the", "an" => "and")

Comment (xvrl) [inline README.md]: fixed

Comment (rasahner) [inline README.md]: thanks, +1

**Comment by nishantmonu51** [comment]
:+1:

**Pull request merged**